src/commands/handler.ts

```diff
@@ -22,7 +22,7 @@ export async function createSpringBootProjectCmdHandler(context: vscode.Extensio
     return;
   }
 
-  await vscode.commands.executeCommand('spring.initializr.maven-project');
+  await vscode.commands.executeCommand('spring.initializr.createProject');
```

Comment:
Need extract the command ID as constant values. Otherwise, it is easy to break the logic.

Reply:
good suggestion, but I don't want to do it in this PR. Suggest to extract all other command names together in another separate PR.